prov/util/src/util_cq.c

```diff
@@ -135,7 +135,7 @@ ssize_t ofi_cq_read(struct fid_cq *cq_fid, void *buf, size_t count)
 	if (count > ofi_cirque_usedcnt(cq->cirq))
 		count = ofi_cirque_usedcnt(cq->cirq);
 
-	for (i = 0; i < count; i++) {
+	for (i = 0; i < (ssize_t)count; i++) {
```

Comment:
for this case - yes, it is possible, re-implemented, but in other cases there is code like ```i = ofi_cq_read(cq_fid, buf, count);``` wher unsigned arithmetic will fail